db/schema.rb

```diff
@@ -507,11 +507,8 @@
 
   create_table "school_admins", force: :cascade do |t|
     t.bigint "user_id"
-    t.bigint "school_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["school_id"], name: "index_school_admins_on_school_id"
-    t.index ["user_id", "school_id"], name: "index_school_admins_on_user_id_and_school_id", unique: true
```

Comment:
Maybe it's now a good idea to ensure that a user has only school_admin entry in the database on similar lines.

Reply:
Yeah, basically updating the index on `user_id` to make it `unique: true`.